src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.headerNav,
+.navList,
+.navLink {
+  display: flex;
+  align-items: center;
+}
+
+.headerNav {
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.navBlock,
+.navIcon {
+  display: block;
+}
+
+.navIcon {
+  width: 40px;
+  height: 40px;
+}
+
+.navList {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.navItem:not(:last-child) {
+  margin-right: 20px;
+}
+
+.navLink {
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  height: 60px;
+  position: relative;
+}
+
+.is-active,
```

Comment:
It's better to use these electors separately. 
.is-active with nesting pseudo-element ::after

.navLink with nesting pseudo-class :hover